app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/NavigationTabs.tsx

```diff
@@ -42,9 +45,14 @@ export const NavigationTabs = ({ formId }: { formId: string }) => {
         setAriaCurrent={true}
       >
         <span className="text-sm laptop:text-base">
-          <DeleteIcon className="inline-block h-7 w-7" /> {t("responses.status.deleted")}
+          <DeleteIcon className="inline-block size-7" /> {t("responses.status.deleted")}
         </span>
       </TabNavLink>
+      {ability?.can("update", "FormRecord", formId) && (
```

Comment:
This will always return `true` if a user is logged in.  The `ability.can()` accepts roughly the same format as the `checkPrivilege()`.  The 3rd agrument of `ability.can` is for the `field`.  In this case we're checking if a user can `update` the `FormRecord` field `${formId}` which resolves to true because a user can generaly modify all fields on a "FormRecord" except for `isPublished`

Reply:
We can chat more about what you'd like to accomplish here with a privilege check and brainstorm.

Reply:
Removed the check for now, as it is not doing what was intended. The page itself provides the security needed for the moment- only owners or admins can get here.

In future we'll want to control by user role or however this develops.

https://github.com/cds-snc/platform-forms-client/pull/4392/commits/1e3d7c29e7a5c1182fe31938b5f9685dd6e97fad